### DISCUSSION TITLE
Change set biome effect

### DISCUSSION THREAD

**Discussion by JonesJugHead** [SkriptLang/Skript#5631] [Polls]
Hello,

The "set biome %locations%'[s] biome to %biome%" should use the new way that minecraft uses in 1.19.4 with the command /fillbiome

It would allow to have the biomes instantly without having to reboot the server

**Comment by TheLimeGlass** [comment]
The biomes are set instantly, it's the client that needs to leave the area and come back for it to take effect. I'm not home so I won't be able to figure out what vanilla Minecraft does, but Spigot does not have a method in the Block interface to refresh the chunks to the player. It would be an enhancement feature.
